src/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.sol

```diff
@@ -97,7 +97,7 @@ contract FM_BC_Bancor_Redeeming_VirtualSupply_v1 is
 
     using SafeERC20 for IERC20;
 
-    //--------------------------------------------------------------------------
+    // -------------------------------------------------------------------------
```

Comment:
Is this 80 characters in total with whitespace included? if yes just ignore, otherwise that's what it should be. 
its fine if you don't want to do it for the whole file though, if it's more than just the stuff you changed. we can tackle that in a different pr.